package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Ambianic UI is the main user interface to Ambianic Edge deployments.",
   "author": "Ivelin Ivanov",
   "scripts": {
+    "prepare": "husky install",
```

Comment:
@wane-nwani this is the part that was missing. Husky install has to be run in order for git to know that it should be executing the husky pre-commit script on `git commit`. Without this step, git does not know about the `.husky/pre-commit` script.

As you can see, I also added an `npm prepare` step to `.gitpod.yml` so that developers working on ambianic-ui via gitpod get the pre-commit script installed automatically.

For developers running in other environments, I added a section in the README document about running `npm run prepare` right after `npm install`.

Reply:
@vickywane not sure if you missed the comment above because you submitted the PR with a different account. Waiting on your approval to proceed with merge.

Reply:
Hello @ivelin 

Thanks for the script added. I'll pull your changes locally and test them. 

In the meantime, you have my approval to merge this pull request.

Reply:
OK. Thanks.